ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +453,12 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
```

Comment:
>  Why don't we try to deal with this simple case, unless anyone thinks that it really should leak.

My goal with adding a second, more complex test is so that once we fix the original, we can also verify that this second one also passes. It's possible that there's a separate leak!